deploy/basic/terraform/policies.tf

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# 
+
+# Create lifecycle policy to delete temp files
+resource "oci_objectstorage_object_lifecycle_policy" "mushop_deploy_assets_lifecycle_policy" {
+  namespace = data.oci_objectstorage_namespace.user_namespace.namespace
+  bucket    = oci_objectstorage_bucket.mushop.name
+
+  rules {
+    action      = "DELETE"
+    is_enabled  = "true"
+    name        = "mushop-delete-deploy-assets-rule"
+    time_amount = "1"
+    time_unit   = "DAYS"
+  }
+  depends_on = [oci_identity_policy.mushop_basic_policies, oci_objectstorage_object.mushop_wallet]
+}
+
+# Create policies for MuShop based on the features
+resource "oci_identity_policy" "mushop_basic_policies" {
+  name           = "mushop-basic-policies-${random_string.deploy_id.result}"
+  description    = "Policies created by terraform for MuShop Basic"
+  compartment_id = var.compartment_ocid
+  statements = (var.use_encryption_from_oci_vault ?
+    (var.create_vault_policies_for_group ? (
+      concat(local.allow_object_storage_lifecycle_statement, local.allow_object_storage_service_keys_statements, local.allow_group_manage_vault_keys_statements)
+      ) : concat(local.allow_object_storage_lifecycle_statement, local.allow_object_storage_service_keys_statements)
+  ) : local.allow_object_storage_lifecycle_statement)
+  freeform_tags = local.common_tags
+}
+
+locals {
```

Comment:
Same comment about `locals`

Reply:
Here are the "policy" locals that only serve the policies.tf

Reply:
locals moved per recommendation